### ISSUE TITLE
Issues with logout

### ISSUE THREAD

**Issue by soend** [nahidf/auth0-adventures#1]
Hei, when playing around with blazor and auth0 did u notice any issues with logging out? While i was testing i noticed that at some random times logout just doesnt work.

**Comment by nahidf** [comment]
Not that I recall really. How can I replicate it? just login/logout? @soend

**Comment by soend** [comment]
> Not that I recall really. How can I replicate it? just login/logout? @soend

Yes, just log in and logout and randomly it will say logged out but in the header still shows username and logout button. Also for some reason logout seems to work every time when you manually refresh the page while u are logged in and then click log out.

[Screenshot](https://i.imgur.com/mFp9Rzq.png)

**Comment by soend** [comment]
I played around with it bit more and got some good results.. [Here i posted what i did](https://community.auth0.com/t/securing-blazor-webassembly-apps/46661/64?u=soend). Maybe if you got time would be good if you could also test this if this solution works for you.
